src/main/groovy/io/saagie/plugin/dataops/models/ListVariableContainer.groovy

```diff
@@ -0,0 +1,7 @@
+package io.saagie.plugin.dataops.models
+
+class ListVariableContainer {
+	String projectId
+	String scope
+	ArrayList listVariableByNameAndScope
```

Comment:
In english, we don't say "list variable by name and scope" but "variableListByNameAndScope"